.github/workflows/kind.yml

```diff
@@ -490,12 +490,9 @@ jobs:
         curl -Lo ./kind https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/kind-$(uname)-amd64
         chmod +x ./kind
         sudo mv kind /usr/local/bin
-    - name: Build netpol Docker image
-      working-directory: hack/netpol
-      run: make
     - name: Run netpol tests
-      working-directory: hack/netpol
-      run: ./test-kind.sh
+      working-directory: ci/kind
+      run: ./test-np.sh
```

Comment:
where is test-np.sh?

Reply:
Added the file

Reply:
netpol is a more well-known short name of networkpolicy,  can we name the script test-netpol.sh?